frontend/src/app/modules/boards/boards-sidebar/boards-menu.component.ts

```diff
@@ -33,12 +33,22 @@ export class BoardsMenuComponent {
               private readonly boardCache:BoardCacheService,
               private readonly currentProject:CurrentProjectService,
               private readonly mainMenuService:MainMenuNavigationService) {
+  }
 
+  ngOnInit() {
     // When activating the work packages submenu,
     // either initially or through click on the toggle, load the results
     this.mainMenuService
       .onActivate('board_view')
-      .subscribe(() => this.boardService.loadAllBoards());
+      .subscribe(() => {
+        this.focusBackArrow();
+        this.boardService.loadAllBoards()
+      });
+  }
+
+  private focusBackArrow() {
+    let buttonArrowLeft = jQuery('*[data-name="board_view"] .main-menu--arrow-left-to-project');
```

Comment:
Just nitpicking, I think you don't need the `*` here, the attribute selector should work fine on its own